ios-starter/objc-starter/FriendlyChatObjC.xcodeproj/project.pbxproj

```diff
@@ -116,16 +123,36 @@
 			path = assets;
 			sourceTree = "<group>";
 		};
+		C8B7C93207D6EDA225C6A35E /* Frameworks */ = {
+			isa = PBXGroup;
+			children = (
+				2487A9E34B9A5F1CBE3A30DF /* libPods-FriendlyChatObjC.a */,
+			);
+			name = Frameworks;
+			sourceTree = "<group>";
+		};
+		C972C5F323D11B7398E70AD9 /* Pods */ = {
+			isa = PBXGroup;
+			children = (
+				BA5E624716EDE53F89F10214 /* Pods-FriendlyChatObjC.debug.xcconfig */,
+				D576AA3FCD48E8CC5B68227B /* Pods-FriendlyChatObjC.release.xcconfig */,
+			);
+			name = Pods;
+			sourceTree = "<group>";
+		};
 /* End PBXGroup section */
 
 /* Begin PBXNativeTarget section */
 		1079607F1C6130EF003DB6CC /* FriendlyChatObjC */ = {
 			isa = PBXNativeTarget;
 			buildConfigurationList = 107960A21C6130EF003DB6CC /* Build configuration list for PBXNativeTarget "FriendlyChatObjC" */;
 			buildPhases = (
+				A9C34ECFB4DDCA6F9EAE2B6A /* [CP] Check Pods Manifest.lock */,
 				1079607C1C6130EF003DB6CC /* Sources */,
 				1079607D1C6130EF003DB6CC /* Frameworks */,
 				1079607E1C6130EF003DB6CC /* Resources */,
+				91C2F7EC4501A7E7ED3122EE /* [CP] Embed Pods Frameworks */,
+				12980CAF543D23BCBE2B26DF /* [CP] Copy Pods Resources */,
```

Comment:
please run "pod deintegrate" to delete all references to cocoapods in .xcodeproj